packages/opentelemetry-basic-tracer/src/platform/browser/timer-util.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/** This is Node specific, does nothing in case of browser */
+export function unrefTimer(timer: NodeJS.Timeout): void {}
```

Comment:
In the browser, `timer` is of type `number`. If you change it to that, will this compile?

At some point I'm thinking about trying to make all Node typings in the `core` and `basic-tracer` packages be confined to Node platform code so that TypeScript users of the library in the browser wouldn't need Node typings, but I'll look into that later on. Anyway, changing this to `number` now could help that effort in the future.

Reply:
> In the browser, timer is of type number. If you change it to that, will this compile

Yes, possible. :)